modules/imgproc/src/resize.cpp

```diff
@@ -1977,14 +1977,14 @@ struct VResizeLinear<uchar, int, short, FixedPtCast<int, uchar, INTER_RESIZE_COE
         #if CV_ENABLE_UNROLLED
         for( ; x <= width - 4; x += 4 )
         {
-            dst[x+0] = uchar(( ((b0 * (S0[x+0] >> 4)) >> 16) + ((b1 * (S1[x+0] >> 4)) >> 16) + 2)>>2);
-            dst[x+1] = uchar(( ((b0 * (S0[x+1] >> 4)) >> 16) + ((b1 * (S1[x+1] >> 4)) >> 16) + 2)>>2);
-            dst[x+2] = uchar(( ((b0 * (S0[x+2] >> 4)) >> 16) + ((b1 * (S1[x+2] >> 4)) >> 16) + 2)>>2);
-            dst[x+3] = uchar(( ((b0 * (S0[x+3] >> 4)) >> 16) + ((b1 * (S1[x+3] >> 4)) >> 16) + 2)>>2);
+            dst[x+0] = uchar((( ((b0 * (S0[x+0] >> 4)) + (b1 * (S1[x+0] >> 4))) >> 16) + 2)>>2);
```

Comment:
The problem in #26118 is with `INTER_LINEAR_EXACT`. These changes are irrelated.

Reply:
Helo @fengyuentau ,
The initial changes were applied when the condition on line 3932 was met. Following that, further improvements were made to enhance the precision of calculations within the `else` block as well.
![S__41074767](https://github.com/user-attachments/assets/a9a63599-0bd7-4e27-80ac-cba384f835c8)

I have made the necessary modifications to the INTER_LINEAR_EXACT implementation. Specifically, I have improved the calculation accuracy by utilizing higher precision during the interpolation process to minimize rounding errors. 

Thank you,
Alvis